portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,94 @@
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <!--------header------->
+      <header>
+          <a class="logo"><h1>Yifan</h1></a>
+          <ul class="navlist">
+            <li><a href="#home" class="btn">Home</a></li>
+            <li><a href="#about" class="btn">About Me</a></li>
+            <li><a href="#project" class="btn">Projects</a></li>
+          </ul> 
+      </header>
+    
+      <!--------home------->
+      <section class="home" id="home">
+        <div class="home-text">
+            <h3>Hello I'm</h3>
+            <h1>Yifan Sun</h1>
+            <h5>A Computer Science student <span>From UBC</span></h5>
+            <p>Currently working as a Software Developer Intern <span>at Unity</span></p>
+            <div class="social">
+                <a href="https://github.com/xiaoalem"><i class='bx bxl-github'></i></a>
+                <a href="https://www.linkedin.com/in/yfsunubc/"><i class='bx bxl-linkedin' ></i></a>
+                <a href="mailto:sueyifankk@gmail.com"><i class='bx bx-envelope'></i></a>
+            </div>
+        </div>
+
+        <div class="home-image">
+            <img src="images/lake_louis.jpg" style="width: auto; height: auto;max-width: 540px;max-height: 540px">
+        </div>
+      </section>
+
+      <!---------About-------->
+      <section class="about" id="about">
+          <div class="about-img">
+              <img src="images/winter.jpg" style="width: auto; height: auto;max-width: 540px;max-height: 540px">
+          </div>
+
+          <div class="about-text">
+              <h3>I'm a Junior Software Developer located in Vancouver</h3>
+              <h2>I'm very passionate to software and data engineering</h2>
+              <p>In addition to the internships at Unity and Enviornment Canada, I have experience with geospatial analysis and hold a Master of Forestry degree from UBC.</br>
+              I love to expand my passion with your company, team and you.</p>
+              <div id="greeting-container"></div>
+          </div>
+          
+          <p>I'm a mountain lover🏔🗻, click <a href="gallary.html">here</a> to see more mountains.</p>
+          <p>Click here to know more about: <button onclick="addRandomGreeting()">ME</button>
+        </p>
+          
+      </section>
+
+      <!-----------Project---------->
+      <section class="project" id="project">
+          <div class="heading">
+              <h2>Projects</h2>
+          </div>
+          <div class="project-content">
+            <div class="col">
+                <div class="col-img">
+                  <img src="images/winter.jpg" style="width: auto; height: auto;max-width: 540px;max-height: 540px">
```

Comment:
The image size seems to consistent across multiple images. Might be  a good idea to move to a css class.